src/interfaces/iTypologyExpression.ts

```diff
@@ -11,7 +11,7 @@ export interface IRuleValue {
 export interface IRule {
   id: string;
   cfg: string;
-  ref: string;
+  ref?: string;
```

Comment:
why nullable?

Reply:
Ref property can potentially be removed as none of our live data have ref values under the terms object. But didn't want to remove if it served a purpose.
eg.

```
"expression": {
    "operator": "-",
    "terms": [
      {
        "id": "000@1.0.0",
        "cfg": "1.0.0"
      },
      {
        "id": "999@1.0.0",
        "cfg": "1.0.0"
      }
   ]
}